packages/core/cms/faststore/content-types.json

```diff
@@ -100,90 +110,84 @@
-                            },
-                            "label": {
-                              "title": "Label",
-                              "type": "string",
-                              "default": "Pickup Anywhere"
-                            }
-                          }
+                          "default": "Pickup Anywhere"
                         }
                       }
-                    },
-                    "regionSlider": {
-                      "title": "RegionSlider SlideOver",
+                    }
+                  }
+                },
+                "regionSlider": {
+                  "title": "RegionSlider SlideOver",
+                  "type": "object",
+                  "properties": {
+                    "title": {
+                      "title": "RegionSlider title",
                       "type": "object",
                       "properties": {
-                        "title": {
-                          "title": "RegionSlider title",
-                          "type": "object",
-                          "properties": {
-                            "setLocation": {
-                              "title": "Set Location title",
-                              "type": "string",
-                              "default": "Set Location"
-                            },
-                            "changeLocation": {
-                              "title": "Change Postal Code title",
-                              "type": "string",
-                              "default": "Change Location"
-                            },
-                            "changePickupPoint": {
-                              "title": "Change Pickup Point location title",
-                              "type": "string",
-                              "default": "Change Store"
-                            }
-                          }
-                        },
-                        "pickupPointChangeApplyButtonLabel": {
-                          "title": "Change Pickup Point apply button label",
+                        "setLocation": {
+                          "title": "Set Location title",
                           "type": "string",
-                          "default": "Update"
+                          "default": "Set Location"
                         },
-                        "choosePickupPointAriaLabel": {
-                          "title": "Choose Pickup Point aria-label (radio group)",
+                        "changeLocation": {
+                          "title": "Change Postal Code title",
                           "type": "string",
-                          "default": "Choose a store"
+                          "default": "Change Location"
                         },
-                        "noPickupPointsAvailableInLocation": {
-                          "title": "No Pickup Point available near location message",
+                        "changePickupPoint": {
```

Comment:
🎗️ If merging this PR after https://github.com/vtex/faststore/pull/2890
We had introduced `globalChangePickupPoint`. So, remember to adapt it if necessary 👍